### PR TITLE
doc(README.md): modify short description of the project

### PR DESCRIPTION
**Pull request by arschles**
This patch is a documentation update that is part of https://github.com/orgs/deis/projects/1#card-732251

cc/ @krancour

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,10 +3,11 @@
 [![Build Status](https://travis-ci.com/deis/steward-cf.svg?token=UQsxfwHAz3NPyVqxkrrp&branch=master)]
 (https://travis-ci.com/deis/steward-cf)
 
-Steward CF is a Kubernetes "service catalog controller--" a Kubernetes native service broker
-gateway for service brokers that implement the [CloudFoundry Service Broker API][cfbroker]. Steward
-CF is built on top of the [Steward](https://gihub.com/deis/steward-framework) service catalog
-controller framework.
+Steward CF is a Kubernetes service catalog controller implemantation that
```
Comment (krancour) [inline README.md]: spelling: implemantation --> implementation

File: README.md [hunk 2]
```diff
@@ -3,10 +3,11 @@
...
+translates standard service catalog Kubernetes resources into [CloudFoundry Broker
+API][cfbroker]. This project is built on top of
```
Comment (sgoings) [inline README.md]: into CloudFoundry Broker API ____.

File: README.md [hunk 3]
```diff
@@ -3,10 +3,11 @@
 [![Build Status](https://travis-ci.com/deis/steward-cf.svg?token=UQsxfwHAz3NPyVqxkrrp&branch=master)]
 (https://travis-ci.com/deis/steward-cf)
 
-Steward CF is a Kubernetes "service catalog controller--" a Kubernetes native service broker
-gateway for service brokers that implement the [CloudFoundry Service Broker API][cfbroker]. Steward
-CF is built on top of the [Steward](https://gihub.com/deis/steward-framework) service catalog
-controller framework.
+Steward CF is a Kubernetes service catalog controller implemantation that
+translates standard service catalog Kubernetes resources into [CloudFoundry Broker
```
Comment (krancour) [inline README.md]: Suggest moving "Kubernetes" before "service catalog." Think it reads better.

**Comment by krancour** [comment]
Fine with this after the edits I suggested.

File: README.md (hunk 1 above)
Comment (arschles) [reply README.md]: Fixed in 14dfbf4cafcf7ab642fefaf642aa509b9abd8faa

File: README.md (hunk 2 above)
Comment (arschles) [reply README.md]: added "calls" in 0d68eb48dbb6e3012fbeabb1b997ee2e4c7b77f5

File: README.md (hunk 3 above)
Comment (arschles) [reply README.md]: good idea. fixed in 0797833a660a02910809a0881d35395bcb48a267

**Pull request merged**